restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java

```diff
@@ -795,7 +798,7 @@ private void onReceiveTimeout(Object message) throws TransitionFailedException,
             fsm.transition(message, finishDialing);
         } else if (is(bridging)) {
             fsm.transition(message, finishDialing);
-        } else if (is(playing)) {
+        } else if (is(playing) || is(initializingCall)) {
```

Comment:
is this a new transition we didnt covered? which scenario are we covering here (timeout during initializing)?

Reply:
during `initializingCall` we are starting a new timer, so if timer expires and we are still in `initializingCall ` phase then VI will move onto `Finished` state. JFYI we already do the same for Busy state.

Reply:
during `initializingCall` we are starting a new timer, so if timer expires and we are still in `initializingCall ` phase then VI will move onto `Finished` state. JFYI we already do the same for Busy state.